src/components/invite/Invite.js

```diff
@@ -282,6 +283,10 @@ const Invite = () => {
     setShowHowTo(!showHowTo)
   }
 
+  useEffect(() => {
```

Comment:
could be included to the useInviteScreenOpened

Reply:
done